id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoDraftServiceImpl.java

```diff
@@ -274,6 +274,7 @@ private void updateDemographicData(IdRequestDTO request, UinDraft draftToUpdate)
 			Configuration configuration = Configuration.builder().jsonProvider(new JacksonJsonProvider())
 					.mappingProvider(new JacksonMappingProvider()).build();
 			DocumentContext inputData = JsonPath.using(configuration).parse(requestDTO.getIdentity());
+			((LinkedHashMap) ((JsonContext) inputData).json()).put("UIN", null);
```

Comment:
why you are putting UIN as null for inputData

Reply:
removed